.travis.yml

```diff
@@ -16,6 +16,7 @@ branches:
   - development
   - qa
   - master
+  - SCC-2143
```

Comment:
Why this? I don't think you need this to have it build against a PR involving that branch.

Reply:
I'm pretty sure it did not build until I added that. Most of our other travis configs do not have this. Thoughts on removing this property?

Reply:
Like here: https://github.com/NYPL/hold-request-service/pull/39/checks

Reply:
Odd. It should have built (because a PR against development is technically a development build - see https://docs.travis-ci.com/user/pull-requests/ ). But you're right that it's not necessary. Remove it please